### PR TITLE
Add mpl-image-labeller

### PR DESCRIPTION
**Pull request by ianhi**
## PR Summary

Add my newest interactive mpl package. For labelling images for clsasification models

### CODE REVIEW

File: packages/mpl-image-labeller.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+repo: ianhi/mpl-image-labeller
+keywords: image, classification, label, labeler, labeller
+section: interactivity
+description: Easily (label) annotate images with for training classification models.
```
Comment (jklymak) [inline packages/mpl-image-labeller.yml:4]: - can you make the sentence grammatical?  Also, this makes it sound like you are annotating the image, which is not _really_ what you are doing, is it?

**Comment by ianhi** [comment]
I think the new version is clearer? And certainly with fewer grammatical errors - sorry about that

**Comment by ianhi** [comment]
unclear to me if that build failure is my fault?

**Comment by jklymak** [comment]
You probably need to rebase to get a modern setup.

**Pull request merged**